Policies/Kubernetes/container-allowed-capabilities/rego/src_test.rego

```diff
@@ -0,0 +1,263 @@
+package k8sazureallowedcapabilities
+
+test_input_all_allowed {
+    input := { "review": input_review([cadd(["one", "two"])]), "parameters": {"allowedCapabilities": ["*"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_all_allowed_container_x2 {
+    input := { "review": input_review([cadd(["one", "two"]), cadd(["three"])]), "parameters": {"allowedCapabilities": ["*"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_one_allowed {
+    input := { "review": input_review([cadd(["one"])]), "parameters": {"allowedCapabilities": ["one"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_one_allowed_container_x2 {
+    input := { "review": input_review([cadd(["one"]), cadd(["one"])]), "parameters": {"allowedCapabilities": ["one"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_two_allowed_container_x2 {
+    input := { "review": input_review([cadd(["one"]), cadd(["two"])]), "parameters": {"allowedCapabilities": ["one", "two"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_two_allowed_two_used_container_x2 {
+    input := { "review": input_review([cadd(["one", "two"]), cadd(["one", "two"])]), "parameters": {"allowedCapabilities": ["one", "two"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+test_input_none_allowed {
```

Comment:
Can you please add below two tests?
1. add capabilities not defined in input object and allowedCapabilities array is also empty
2. add capabilities not defined in input object and allowedCapabilities array is not empty